tests/js/spec/views/releases/list/index.spec.jsx

```diff
@@ -186,7 +192,7 @@ describe('ReleasesList', function () {
       })
     );
 
-    expect(input.prop('value')).toBe('derp');
+    expect(input.prop('value')).toBe('derp ');
```

Comment:
was the space at the end intentional?

Reply:
it seems to be coming from the newer search bar 🤷 automatically adds a space at the end.

Reply:
ahhh gotcha